macosx/CelestiaAppCore.mm

```diff
@@ -22,6 +22,10 @@
 #include "qtcapture.h"
 #import <Carbon/Carbon.h>
 
+// HACK: SetThemeCursor is only available in Carbon.h without LP64 defined.
+//       However it is still present in 64bit system, thus, we just extern it.
+extern "C" OSStatus SetThemeCursor(ThemeCursor inCursor);
```

Comment:
What if it will be removed in the next OSX release? I'd prefer to load if dynamically and only if it's actually present in the system then use it. Otherwise omit calls to SetThemeCursor

Reply:
it is meant to stay for long, since modern ways to change cursor should actually still map to this function in the Carbon framework although Carbon is deprecated since like 6 years ago if I'm not mistaken. anyway I will edit the hack out and adopt the modern method